integration_tests/parameterized/Parameterized.csproj

```
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="sdk\dotnet\Pulumi.Pkg.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <DefaultItemExcludes>$(DefaultItemExcludes);sdk/**/*.cs</DefaultItemExcludes>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

This will need to be updated to match updates to other integration test projects in https://github.com/pulumi/pulumi-dotnet/pull/410

The macos-13 runner doesn't come with .NET 6 installed. So when running the .NET 8 matrix, the integration tests were failing because they target .NET 6. This change allows it to work with only .NET 8 installed, because the GHA workflow sets `TARGET_FRAMEWORK` envvar based on the matrix .NET version.

```suggestion
    <TargetFramework Condition=" '$(TARGET_FRAMEWORK)' != '' ">$(TARGET_FRAMEWORK)</TargetFramework>
    <TargetFramework Condition=" '$(TARGET_FRAMEWORK)' == '' ">net6.0</TargetFramework>
```